src/poetry/console/commands/show.py

```diff
@@ -215,9 +276,22 @@ def handle(self) -> int | None:
                     ),
                 )
 
+                if self.option("why"):
+                    required_by = reverse_deps(locked, locked_repo)
+                    required_by_length = max(
+                        required_by_length, len(" from " + ",".join(required_by.keys()))
+                    )
+
         write_version = name_length + version_length + 3 <= width
         write_latest = name_length + version_length + latest_length + 3 <= width
-        write_description = name_length + version_length + latest_length + 24 <= width
+        write_why = self.option("why") and (
+            name_length + version_length + latest_length + required_by_length + 3
```

Comment:
Maybe put the bits other than 3 in a variable as it's being reused. Nitpick.

Reply:
Not quite sure I understood you correctly; but maybe done :-)